tests/test_gyrovector_math.py

```diff
@@ -696,3 +696,29 @@ def test_weighted_midpoint_weighted_zero_sum(_k, lincomb):
     assert torch.isfinite(mid).all()
     mid.sum().backward()
     assert torch.isfinite(a.grad).all()
+
+def test_mobius_matvec_vs_log_exp_equivalence():
+    """
+    Test: mobius_matvec = expmap0(W logmap0(x))
+    Now works for high dimensions (e.g., 128), previously only stable for small dims (e.g., 20).
+    """
+    from geoopt.manifolds import Stereographic
+    import torch as th
```

Comment:
Both of these packages are imported at the top of the file, let's use those import aliases